prow/overlays/cnv-prod/labels_operate-first.yaml

```diff
@@ -312,6 +378,18 @@ default:
         - name: wontfix
       target: both
       addedBy: humans
+    - color: fc05d7
+      description: Indicates an issue/PR is related to end-user documentation.
+      name: users/documentation
+      target: both
+      prowPlugin: label
+      addedBy: anyone
+    - color: fc05d7
+      description: Indicates an issue/PR is user experience (UX).
+      name: user/experience
```

Comment:
Should this one replace [`sig/user-experience`](https://github.com/operate-first/support/labels/sig%2Fuser-experience)?

Reply:
> Should this one replace [`sig/user-experience`](https://github.com/operate-first/support/labels/sig%2Fuser-experience)?

I've been trying to figure out what `sig` means. I see it in the Prow commands but haven't looked much further as it sounded like it might be about special interest groups, which didn't seem to be the fit.